npm_test.go

```diff
@@ -499,6 +510,53 @@ func TestNpmPackInstall(t *testing.T) {
 	assert.Len(t, npmBuildInfo.Modules, 0)
 }
 
+func TestNpmPackWorkspaces(t *testing.T) {
```

Comment:
please explain what we are testing here, and what its different from the simple npm pack test